kodilla-testing/build.gradle

```diff
@@ -7,8 +7,12 @@ sourceCompatibility = 1.8
 
 repositories {
     mavenCentral()
+    jcenter()
 }
 
+
 dependencies {
+    compile 'com.google.guava:guava:24.1-jre'
     testCompile group: 'junit', name: 'junit', version: '4.12'
+    testCompile "org.mockito:mockito-core:2.+"
```

Comment:
nie uzywaj `+` to powoduje ryzyko ze przy ktoryms buildzie wersja sie zmieni i cos przestanie dzialac

Reply:
zastap mockito przez
testCompile group: 'org.mockito', name: 'mockito-all', version: '1.8.4'

Reply:
testCompile group: 'org.mockito', name: 'mockito-core', version: '2.18.0'